src/test/fuzz/tx_pool.cpp

```diff
@@ -29,6 +29,15 @@ struct MockedTxPool : public CTxMemPool {
     }
 };
 
+class DummyChainState final : public CChainState
+{
+public:
+    CTxMemPool* SetMempool(CTxMemPool* mempool)
```

Comment:
Why does this return a pointer to the `CTxMemPool`, which never gets used. Can it just return void?